### PR TITLE
kubectl: add test coverage for cordon command

### PR DESCRIPTION
**Pull request by omerap12**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Added test coverage for the `kubectl cordon` command.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

**Comment by omerap12** [comment]
cc @ardaguclu

File: staging/src/k8s.io/kubectl/pkg/drain/cordon_test.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+				t.Error("Expected error but got none")
+			}
+			if !tt.expectError && err != nil {
+				t.Errorf("Unexpected error: %v", err)
+			}
+			if !tt.expectError && helper == nil {
+				t.Error("Expected non-nil helper")
+			}
+			if tt.expected != nil && helper != nil {
+				if helper.node.Name != tt.expected.node.Name {
```
Comment (ardaguclu) [inline staging/src/k8s.io/kubectl/pkg/drain/cordon_test.go:85]: I don't think, comparing the node name only would suffice. What about cmp.Diff, etc.?

**Review by ardaguclu — COMMENTED** [review]
Thank you
/triage accepted
/priority backlog

File: staging/src/k8s.io/kubectl/pkg/drain/cordon_test.go (hunk 1 above)
Comment (omerap12) [reply staging/src/k8s.io/kubectl/pkg/drain/cordon_test.go]: You're absolutely right (it wasn't sufficient, as you can see, lol). I addressed it in [f544584](https://github.com/kubernetes/kubernetes/pull/129202/commits/f5445844d218e98f73a11b82dcae7435f09002ea).

**Pull request merged**